content/ember-5-0-released.md

```diff
@@ -0,0 +1,233 @@
+   `npx ember-cli-update --to 4.12`. The
+   [general Ember upgrade guide](https://cli.emberjs.com/release/basic-use/upgrading/#upgradinganemberappitself)
+   has more details about how to upgrade your Ember app/addon.
+2. Make sure your app builds successfully.
+3. Resolve all deprecation warnings. These Deprecated APIs are removed in Ember 5.0. You may need to upgrade some of your addon dependencies if they are using deprecated APIs. 
+See the [Ember Deprecation Guide](https://deprecations.emberjs.com/)
+for more details about specific deprecations and how to resolve them.
+Applications that need to upgrade through several versions may want to
+consider isolating individual deprecations by using
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow).
+4. Make sure your app builds successfully and your test suite passes with no
+   deprecations.
+5. Upgrade your app to Ember 5.0. Again,
+   many developers can do this by running
+   `npx ember-cli-update --to 5.0`. Refer to the
+   [general Ember upgrade guide](https://cli.emberjs.com/release/basic-use/upgrading/#upgradinganemberappitself)
+   for more details about how to upgrade your Ember app/addon.
+
+For help or advice along the way, visit [Ember's forums or chat groups](https://emberjs.com/community/).
+
+<!-- TODO - do we need to mention anything here about htmlbars, fastboot, embroider, or ember-auto-import? -->
+
+## The journey towards Ember Polaris
+
+Polaris is the name of an upcoming edition of Ember, first announced at EmberConf 2021. Over the course of 5.x, new features will be added to Ember that will eventually make up the new edition.
+
+Since our major versions tend to be uneventful, Editions are the moment where the Ember community comes together to learn and build in new ways.
+An edition is a set of features, corresponding documentation, and tools that create a cohesive programming model.
+An edition marks a shift and step forward in the framework, beyond what individual features would accomplish.
+Ember Octane was the first new edition that was added to Ember.
+
+Stay tuned for future blog posts with more details and calls for contributors!
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 5.0
+
+Ember.js 5.0 introduces no new public API. Instead, it is comprised of bug fixes and the removal of previously deprecated public API from the 4.x cycle.
+
+This release drops support for Node 14. Node 14 became end of life (it no longer receives security updates) as of April 2023.
+
+#### Bug Fixes
+
+Ember.js 5.0 introduced 0 bug fixes.
+
+#### Features
+
+Ember.js 5.0 introduced 2 features.
+
+1. Feature description
+2. Feature description
+
+#### Deprecations
+
+Ember.js 5.0 introduced 0 deprecations. It removed all deprecations that were
+introduced before 4.10 and slated for removal in 5.0.
+
+There were only 5 deprecations for Ember 5.0:
```

Comment:
This wording is a little confusing since 5.0 doesn't add the deprecations, as stated above. I propose:

```suggestion
Ember 5.0 removes five features deprecated during 4.x:
```

Reply:
Waaaaay better.